packages/core/src/Stores/traders-hub-store.js

```diff
@@ -72,4 +90,125 @@ export default class TradersHubStore extends BaseStore {
+        this.available_cfd_accounts = all_available_accounts.map(account => {
+            return {
+                ...account,
+                description: account.description,
+                is_visible: ['synthetic', 'all'].some(market_type => account.market_type === market_type)
+                    ? this.isDerivedVisible(account.platform)
+                    : this.isFinancialVisible(account.platform_name),
+                // is_disabled: this.has_cfd_account_error(account.platform),
+            };
+        });
+        this.getAvailableDxtradeAccounts();
+        this.getAvailableMt5Accounts();
+    }
+    getAvailableMt5Accounts() {
+        if (this.is_eu_user) {
+            this.available_mt5_accounts = this.available_cfd_accounts.filter(account =>
+                ['EU', 'All'].some(region => region === account.availability)
+            );
+            return;
+        }
+        this.available_mt5_accounts = this.available_cfd_accounts.filter(
+            account => account.platform === CFD_PLATFORMS.MT5
+        );
+    }
+
+    getAvailableDxtradeAccounts() {
+        if (this.is_eu_user) {
+            this.available_dxtrade_accounts = this.available_cfd_accounts.filter(
+                account =>
+                    ['EU', 'All'].some(region => region === account.availability) &&
+                    account.platform === CFD_PLATFORMS.DXTRADE
+            );
+            return;
+        }
+        this.available_dxtrade_accounts = this.available_cfd_accounts.filter(
+            account => account.platform === CFD_PLATFORMS.DXTRADE
+        );
+    }
+    hasAccount(cfd_platform, landing_company_short) {
+        const current_list_keys = Object.keys(this.root_store.modules.cfd.current_list);
+        return current_list_keys.some(key =>
+            landing_company_short
+                ? key.startsWith(`${cfd_platform}.${this.selected_account_type}`)
+                : key.startsWith(`${cfd_platform}.${this.selected_account_type}.${landing_company_short}`)
+        );
+    }
+    isDerivedVisible(platform) {
+        const client = this.root_store.client;
+        // Hiding card for logged out EU users
+
+        if ((!client.is_logged_in && client.is_eu_country) || (client.is_eu && this.hasAccount(platform, 'synthetic')))
+            return false;
+
+        return (
+            isLandingCompanyEnabled({ landing_companies: client.landing_companies, platform, type: 'gaming' }) ||
+            !client.is_logged_in
+        );
+    }
+    isFinancialVisible(platform) {
+        const client = this.root_store.client;
+        return (
+            !client.is_logged_in ||
+            isLandingCompanyEnabled({
+                landing_companies: client.landing_companies,
+                platform,
+                type: 'financial',
+            })
+        );
+    }
+    checkForExistingAccounts(platform, market_type) {
+        const { modules } = this.root_store;
+        const { current_list } = modules.cfd;
```

Comment:
const { modules } = this.root_store;
        const { current_list } = modules.cfd;

        const { current_list } = this.root_store.modules.cfd